src/common/string.cpp

```diff
@@ -0,0 +1,70 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  Utility templates and functions for working with strings.
+ */
+
+#include <cstdarg>
+
+#include <memory>
+#include <system_error>
+
+#include "src/common/string.h"
+
+namespace Common {
+namespace String {
+
+std::string format(const char* format, ...) {
```

Comment:
That should be `const char *format`. Same in `formatV()` and in the .h file too

Reply:
If that's my only asterisk mistake, I'll gladly take it ;)